modules/client/src/multisig/interfaces.ts

```diff
@@ -135,18 +136,20 @@ type MultisigProposalBase = {
   status: ProposalStatus;
   startDate: Date;
   endDate: Date;
+  approvals: string[];
```

Comment:
does this contain the list of addresses who approved?

Reply:
yes